tests/transforms/test_clip_transform.py

```diff
@@ -44,79 +62,111 @@ def setUp(self):
             269,
             49407,
         ]
-        self.bos_token = self.text1_tokens[0]
-        self.eos_token = self.text1_tokens[-1]
-        self.text1_token_len = len(self.text1_tokens)
-        bpe_file = "clip_vocab.bpe"
-        self.bpe_merges_file = get_asset_path(bpe_file)
-        self.clip_transform = CLIPTransform(text_bpe_merges_path=self.bpe_merges_file)
-
-    def test_clip_single_transform(self):
-        transformed_image, transformed_text = self.clip_transform(
-            image=self.image1, text=self.text1
-        )
+
+    @pytest.fixture()
+    def bos_token(self, text1_tokens):
```

Comment:
why are bos_token, eos_token and token lens fixtures. these can be derived from the input?

Reply:
I mostly just did a 1:1 mapping from instance variables defined in `setUp` to fixtures. But I can change these